src/common/column_matrix.h

```diff
@@ -141,28 +134,26 @@ class DenseColumn: public Column<BinIdxType> {
 class ColumnMatrix {
```

Comment:
I think the column matrix can use some cleanups, but for now, I will leave it as it's to avoid regression in performance.